xrmocap/ops/bottom_up_association/matching/base_matching.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import numpy as np
+from typing import Tuple, Union
+from xrprimer.utils.log_utils import get_logger
+
+
+class BaseMatching:
+
+    def __init__(self,
+                 logger: Union[None, str, logging.Logger] = None) -> None:
+        """Base class for association matching.
+
+        Args:
+            logger (Union[None, str, logging.Logger], optional):
+                Logger for logging. If None, root logger will be selected.
+                Defaults to None.
+        """
+        self.logger = get_logger(logger)
```

Comment:
We'd better record convention of bottom-up keypoints in `self.convention`, or pass Keypoints instance in `__call__()`.